onadata/libs/tests/serializers/test_merged_xform_serializer.py

```diff
@@ -424,3 +445,20 @@ def test_matching_fields_by_type(self):
         }  # yapf: disable
 
         self.assertEqual(survey.to_json_dict(), expected)
+
+    def test_merged_dataset_dict_contains_no_bind_attributes(self):
+        """
+        Test get_merged_xform_survey(): should not contain bind elements.
+        """
+        self.project = get_user_default_project(self.user)
+        xform1 = self._publish_markdown(GROUP_A_MD, self.user, id_string='a')
+        xform2 = self._publish_markdown(GROUP_B_MD, self.user, id_string='b')
+        xform3 = self._publish_markdown(GROUP_C_MD, self.user, id_string='c')
+        survey = get_merged_xform_survey([xform1, xform2, xform3])
+
+        result = survey.to_json_dict()
+        count = [child for child in result["children"] if 'bind' in child]
```

Comment:
Should this be a number? i.e.:
``python
count = len([child for child in result["children"] if 'bind' in child])
... 
self.assertEqual(count, 0)
````

Reply:
Made this change.